diagnostic.md

```diff
@@ -35,21 +35,22 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
+ This is an absolute path.
 
- <!-- Answer Ends Here -->
+ The absolute path contains the root directory and all other subdirectories in
+ which a file or folder is contained.
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
- <!-- Answer Starts Here -->
+ It is a relative path.
```

Comment:
Actually, anything on the internet would be an absolute path. A file on the internet can be access by any person from anywhere, using exactly the same address. That's why it's "absolute."